### DISCUSSION TITLE
function

### DISCUSSION THREAD

**Discussion by Lotus6** [projectdiscovery/katana#580] [Q&A]
Add a feature to send each request to Burpsuite for more manual testing

**Comment by ehsandeep** [comment]
@Lotus6 you can use proxy option`katana -proxy http://127.0.0.1:8080`

```console
   -proxy string  http/socks5 proxy to use
```

**Comment by dogancanbakir** [comment]
This discussion closed automatically due to inactivity. Feel free to reopen or start new if still relevant.
